index.html

```diff
@@ -241,27 +241,28 @@ <h3 class="frameworks">Skills</h3>
         </div>
     </section>
 
-    <section class="contact-form">
+    <section class="contact-form" id="Contact">
         <p class="footer-text">
             I'm always interested in hearing about new projects, so if you'd like to
             chat please get in touch.
         </p>
         <form class="form-content" action="https://formspree.io/f/mvonwygw" method="POST">
-            <input type="text" name="full_name" maxlength="30" placeholder="Full name" required />
 
             <div class="first-last">
```

Comment:
- [ ] Great job in implementing the changes requested by the previous reviewer, everything is working as it should, however, on the mobile version, the `full-name` section is not focusable which means its hidden on the mobile view
- [ ] Possible solution would be to remove the display properties in `.first-last` class on mobile view and adjust accordingly. :slightly_smiling_face: 
- [ ] As mentioned by the previous reviewer, Please ensure that your design matches the given design, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Figma</td>
    </tr>
    <tr>
      <td>

 
![Student Form](https://github.com/okemboi/portfolio/assets/43528594/f290ef83-bfcb-4536-81ec-73f1ad98fe30)
 
</td>
      <td>

![Figma Form](https://github.com/okemboi/portfolio/assets/43528594/0eb8ca3d-f527-446d-99bc-51cdd8a0459e)

</td>
    </tr>
  </table>

Reply:
> - [ ] I observed that the previous request made by the code reviewer has not been implemented yet. Specifically, on the mobile version, the full-name section remains non-focusable, indicating that it is still hidden in the mobile view. To address this, the suggested solution is to remove the display properties in the .first-last class for the mobile view and make the necessary adjustments.😊
> 
> - [ ] Additionally, I noticed that your current design still does not align with the given design requirements. Aligning your design with the provided design specifications is crucial as it enhances the likelihood of creating a successful and comprehensive design solution that fulfills user needs, adheres to regulations, and operates efficiently. I kindly urge you to take these recommendations into consideration and make the necessary updates accordingly.👍
> 
> Figma design:
> 
> ![at](https://user-images.githubusercontent.com/43528594/254711062-0eb8ca3d-f527-446d-99bc-51cdd8a0459e.png)
> 
> Your design:
> 
> ![at](https://github.com/okemboi/portfolio/assets/40009354/7d38dc12-3ea2-491e-9a67-2d35a0ec9146)
>